crates/anvil/src/eth/api.rs

```diff
@@ -1852,6 +1856,83 @@ impl EthApi {
         Ok(())
     }
 
+    /// Deals ERC20 tokens to a address
+    ///
+    /// Handler for RPC call: `anvil_dealERC20`
+    pub async fn anvil_deal_erc20(
+        &self,
+        address: Address,
+        token_address: Address,
+        balance: U256,
+    ) -> Result<()> {
+        node_info!("anvil_dealERC20");
+
+        sol! {
+            #[sol(rpc)]
+            contract IERC20 {
+                function balanceOf(address target) external view returns (uint256);
+            }
+        }
+
+        let calldata = IERC20::balanceOfCall { target: address }.abi_encode();
+        let tx = TransactionRequest::default().with_to(token_address).with_input(calldata.clone());
+        let block_number = BlockId::from(self.block_number()?.to::<u64>());
+        let access_list_result = self.create_access_list(WithOtherFields::new(tx), None).await?;
+        let access_list = access_list_result.access_list;
+        println!("> Access list contains {} entries", access_list.0.len());
```

Comment:
these pirntln are useful for debugging but let's removing them before we merge it